src/Microsoft.AspNetCore.Razor.Evolution/DefaultRazorIRLoweringPhase.cs

```diff
@@ -181,11 +221,26 @@ public override void VisitStatementSpan(StatementChunkGenerator chunkGenerator,
 
             public override void VisitMarkupSpan(MarkupChunkGenerator chunkGenerator, Span span)
             {
+                if (span.Symbols.Count == 1 &&
+                    span.Symbols[0] is HtmlSymbol symbol &&
```

Comment:
Bajillion pranav points. On another note though we should stay away from C# 7 until we move to it as a team 😄